src/Compilers/Core/Portable/Collections/Rope.cs

```diff
@@ -86,6 +88,23 @@ private sealed class StringRope : Rope
             private readonly string _value;
             public StringRope(string value) => _value = value;
             public override string ToString() => _value;
+
+            public override string ToString(int maxLength)
```

Comment:
I think it's a GitHub bug with displaying complex diff between two branches. I don't think you'd changed this file, right?
Consider pinging GitHub support and filing a bug.

Reply:
This change is from https://github.com/dotnet/roslyn/pull/63448.

Reply:
Ah, makes sense. Thanks